thoth/messaging/__init__.py

```diff
@@ -55,6 +56,7 @@
     UnresolvedPackageMessage,
     UnrevsolvedPackageMessage,
     UpdateProvidesSourceDistroMessage,
+    KebechetRunUrl,
```

Comment:
please maintain alphabetical order :) it is more readable and we can spot something wrong easier and faster if this list will become very long

Reply:
Thanks